docs/guides/get-started.md

```diff
@@ -0,0 +1,208 @@
+---
+page_title: "Get started with Astro Terraform Provider"
+---
+
+# Get started with Astro Terraform Provider
+In this guide, we will automate the onboarding of a new team by creating and managing an Astro Workspace and Deployment. By the end of this tutorial, you will have a fully automated setup that is reproducible and easily scalable to more teams.
+
+## Step 1: Create Your Terraform Working Directory
+1. Create a folder, `my-data-platform` for your Terraform project.
+2. Save the following code in a file named `terraform.tf`:
+```
+terraform {
+  required_providers {
+    astro = {
+      source  = "astronomer/astro"
+      version = "0.3.0"
+    }
+  }
+}
+
+provider "astro" {
+  organization_id = <your-organization-id>
+}
+```
+3. Insert your organization's ID for `<your-organization-id>`. The working directory will contain all your Terraform code, and all Terraform commands will be run from this directory.
+
+## Step 2: Initialize the Terraform Working Directory
+1. Run `terraform init`. You will see Terraform downloading and installing the Astro Terraform provider to your local computer:
+```
+$ terraform init
+
+Initializing the backend...
+
+Initializing provider plugins...
+- Finding astronomer/astro versions matching "0.3.0"...
+- Installing astronomer/astro v0.3.0...
+- Installed astronomer/astro v0.3.0 (signed by a HashiCorp partner, key ID F5206453FDEA33CF)
+
+...
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+```
+2. The versions and hashes of providers are stored in a generated file `.terraform.lock.hcl`. Store this file in version control.
+
+## Step 3: Authenticate with Astro
+1. [Create an API token](https://www.astronomer.io/docs/astro/automation-authentication#step-1-create-an-api-token) in Astro. Since you are creating a Workspace, you need an [Organization API token](https://www.astronomer.io/docs/astro/organization-api-tokens) with [Organization Owner permissions](https://www.astronomer.io/docs/astro/user-permissions#organization-roles).
+2. Configure the API token as an [environment variable](https://www.astronomer.io/docs/astro/environment-variables) `ASTRO_API_TOKEN` to run Terraform commands:
```

Comment:
its not really an astro environment variable, more so an os environment variable

Reply:
so we probably don't need this url

Reply:
Thanks, link is removed!

Reply:
don't see that change yet? maybe forgot to commit or push

Reply:
Ah yes, I didnt push from my local. Now, they are removed. Thanks